packages/analytics/src/usage-analytics.spec.ts

```diff
@@ -1,5 +1,6 @@
 import os from 'os';
 import nock from 'nock';
+import { expectAllMocksUsed } from '@sap-cloud-sdk/core/test/test-util';
```

Comment:
Meh, do you think it makes sense to put this somewhere else outside of core, e.g. `repo/test-util`?